Xamarin.Forms.Platform.WPF/Renderers/MediaElementRenderer.cs

```diff
@@ -84,22 +47,71 @@ protected override void OnElementChanged(ElementChangedEventArgs<MediaElement> e
 				Control.MediaEnded += Control_MediaEnded;
 				Control.MediaFailed += Control_MediaFailed;
 
+				Element.SeekRequested += Element_SeekRequested;
+				Element.StateRequested += Element_StateRequested;
 				UpdateSource();
 			}
 		}
 
+		private void Element_StateRequested(object sender, StateRequested e)
+		{
+			switch(e.State)
+			{
+				case MediaElementState.Playing:
+					if (Element.KeepScreenOn)
+					{
+						DisplayRequestActive();
+					}
+
+					Control.Play();
+					((IMediaElementController)Element).CurrentState = MediaElementState.Playing;
+					break;
+
+				case MediaElementState.Paused:
+					if (Control.CanPause)
+					{
+						if (Element.KeepScreenOn)
+						{
+							DisplayRequestRelease();
+						}
+
+						Control.Pause();
+						((IMediaElementController)Element).CurrentState = MediaElementState.Paused;
+					}
+					break;
+
+				case MediaElementState.Stopped:
+					if (Element.KeepScreenOn)
+					{
+						DisplayRequestRelease();
+					}
+
+					Control.Stop();
+					((IMediaElementController)Element).CurrentState = MediaElementState.Stopped;
+					break;
+			}
+
+			((IMediaElementController)Element).Position = Control.Position;
```

Comment:
`((IMediaElementController)Element)` quite often used. I think it is possible to make private property `IMediaElementController Controller => Element as IMediaElementController` and use it.